diff -Naur a/fs/proc/task_mmu.c b/fs/proc/task_mmu.c
--- a/fs/proc/task_mmu.c	2024-09-15 16:57:56.000000000 +0200
+++ b/fs/proc/task_mmu.c	2024-09-28 20:58:14.786038495 +0200
@@ -333,6 +333,8 @@
 	dev_t dev = 0;
 
 	if (vma->vm_file) {
+		if(strstr(vma->vm_file->f_path.dentry->d_iname, "libMangoHud.so"))
+			return;
 		const struct inode *inode = file_user_inode(vma->vm_file);
 
 		dev = inode->i_sb->s_dev;
