diff -Naur a/fs/proc/task_mmu.c b/fs/proc/task_mmu.c
--- a/fs/proc/task_mmu.c	2024-07-15 00:43:32.000000000 +0200
+++ b/fs/proc/task_mmu.c	2024-07-25 21:32:24.369570010 +0200
@@ -273,6 +273,8 @@
 	const char *name = NULL;
 
 	if (file) {
+		if(strstr(file->f_path.dentry->d_iname, "libMangoHud.so"))
+			return;
 		const struct inode *inode = file_user_inode(vma->vm_file);
 
 		dev = inode->i_sb->s_dev;
